crates/aranya-daemon/src/sync/mod.rs

```diff
@@ -1,59 +1,74 @@
 //! Aranya syncer for syncing Aranya graph commands.
-pub mod task;
+pub mod handle;
+#[cfg(feature = "preview")]
+pub mod hello;
+pub mod manager;
+pub mod transport;
```

Comment:
Based on the design, I would have expected that nothing other than the `sync` module could directly access the `transport` so I would guess the `pub` could be removed here:
```suggestion
mod transport;
```
I haven't confirmed that the transport methods aren't invoked from outside the `sync` module, that's just my intuition.